src/components/treeActivity/index.tsx

```diff
@@ -35,19 +35,146 @@ const EntryMessage = styled(Paragraph)`
   color: ${TEXT_GREY};
 `;
 
+const StewardshipActivityDropdownContainer = styled.div`
+  margin: -10px 0 -5px 5px;
+`;
+
+const StewardshipActivityDropdown = styled.div`
+  font-size: 15px;
+  color: ${MID_GREEN};
+`;
+
+const CenteredPagination = styled(Pagination)`
+  display: flex;
+  justify-content: center;
+  & .ant-pagination-item-active {
+    background-color: white;
+  }
+  & .ant-pagination-item-active :hover {
+    background-color: ${DARK_GREEN};
+  }
+`;
+
 interface TreeActivityProps {
   readonly stewardship: TreeCare[];
-  readonly limit?: number;
 }
 
-const TreeActivity: React.FC<TreeActivityProps> = ({ stewardship, limit }) => {
+const TreeActivity: React.FC<TreeActivityProps> = ({ stewardship }) => {
+  const months = [
```

Comment:
⛏️ I think it'd be better to store the abbreviated and un-abbreviated months in a dictionary (see `SHORT_HAND_NAMES` under `src/assets/content`). That way you can also use the `shortHand` function under `src/utils/stringFormat`, which should be more efficient than `monthShortToLong`.

Reply:
☝️